docs/source/conf.py

```diff
@@ -66,49 +62,63 @@
 # -- Options for HTML output -------------------------------------------------
 
 # Output file base name for HTML help builder.
-htmlhelp_basename = 'receptorrdoc'
+htmlhelp_basename = "receptorrdoc"
 
 # -- Options for LaTeX output ---------------------------------------------
 
 latex_elements = {
     # The paper size ('letterpaper' or 'a4paper').
     #
     # 'papersize': 'letterpaper',
-
     # The font size ('10pt', '11pt' or '12pt').
     #
     # 'pointsize': '10pt',
-
     # Additional stuff for the LaTeX preamble.
     #
     # 'preamble': '',
-
     # Latex figure (float) alignment
     #
     # 'figure_align': 'htbp',
 }
 
 latex_documents = [
-    (master_doc, 'receptor.tex', 'receptor Documentation',
-     'Red Hat Ansible', 'manual'),
+    (master_doc, "receptor.tex", "receptor Documentation", "Red Hat Ansible", "manual"),
 ]
 
 # -- Options for manual page output ---------------------------------------
 
 # One entry per manual page. List of tuples
 # (source start file, name, description, authors, manual section).
-man_pages = [
-    (master_doc, 'receptor', 'receptor Documentation',
-     [author], 1)
-]
+man_pages = [(master_doc, "receptor", "receptor Documentation", [author], 1)]
 
 # -- Options for Texinfo output -------------------------------------------
 
 # Grouping the document tree into Texinfo files. List of tuples
 # (source start file, target name, title, author,
 #  dir menu entry, description, category)
 texinfo_documents = [
-    (master_doc, 'receptor', 'receptor Documentation',
-     author, 'receptor', 'Overlay network to establish a persistent mesh.',
-     'Miscellaneous'),
+    (master_doc, "receptor", "receptor Documentation", author, "receptor", "Overlay network to establish a persistent mesh.", "Miscellaneous"),
```

Comment:
```suggestion
    (
        master_doc,
        "receptor",
        "receptor Documentation",
        author,
        "receptor",
        "Overlay network to establish a persistent mesh.",
        "Miscellaneous",
    ),
```

Reply:
Just a nit but I'd prefer an expanded format like this for better readability.

Reply:
I tried but black puts it back :-/

Reply:
@thom-at-redhat hmm. which version of black are you using?

When I format with black it actually gives me the expanded list. I wonder if I'm doing something weird.

```
$ black --version
black, 24.4.2 (compiled: yes)
Python (CPython) 3.12.3

$ black docs/source/conf.py

$ grep texinfo_documents -A 9 docs/source/conf.py
texinfo_documents = [
    (
        master_doc,
        "receptor",
        "receptor Documentation",
        author,
        "receptor",
        "Overlay network to establish a persistent mesh.",
        "Miscellaneous",
    ),
```

Reply:
you are correct:
```
$ black --version
black, 24.4.2 (compiled: yes)
Python (CPython) 3.12.3
```
It reformatted correctly the second time, thank you for your help!